pkg/virtctl/vnc/vnc.go

```diff
@@ -155,13 +168,37 @@ func (o *VNC) Run(cmd *cobra.Command, args []string) error {
 			glog.Infof("remote-viewer commandline: %v", args)
 		}
 
-		cmnd := exec.Command("remote-viewer", args...)
+		vncBin := ""
+		osType := runtime.GOOS
+		switch osType {
+		case "darwin":
+			if _, err := os.Stat(TIGER_VNC); err == nil {
+				vncBin = TIGER_VNC
+			}
+			if _, err := os.Stat(CHICKEN_VNC); err == nil {
+				vncBin = CHICKEN_VNC
+			}
+			if _, err := os.Stat(REAL_VNC); err == nil {
```

Comment:
Are we supposted to select the first VNC player or is this deliberately checking for all binaries on the darwin platform?